java/test/TestH5Rref.java

```diff
@@ -0,0 +1,562 @@
+/* * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * *
+ * Copyright by The HDF Group.                                               *
+ * Copyright by the Board of Trustees of the University of Illinois.         *
```

Comment:
I think this is a new file?  If so, it shouldn't have the line or the Board of Trustees.  Also, the support URL 6 lines down should be replaced with " * distribution tree, or in https://www.hdfgroup.org/licenses.               *"

Reply:
It was a split of TestH5R.java

Reply:
Actually all these java test files are new since like 2010.